### PR TITLE
different grids made

### PR DESCRIPTION
**Pull request by Asypher00**
check grids.css for code
Screenshot (18)

### CODE REVIEW

File: components/grids.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+:root{
+    --gridBackground : gray;
+}
+body{
+    font-family: "comic sans ms";
+}
+
```
Comment (nishasen) [inline components/grids.css]: You can define this in one single file, instead of defining it in all pages

File: components/grids.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+.grid-container-two{display: grid;
```
Comment (nishasen) [inline components/grids.css]: start the styles from a new line.

File: components/grids.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 10px;
+}
+.grid-item{
+    background-color: white;
+    border: 2px solid black;
+    padding : 10px;
+    border-radius: 10px;
+}
+
+.grid-container-three{display: grid;
```
Comment (nishasen) [inline components/grids.css]: start the styles from a new line.

File: components/grids.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    font-family: "comic sans ms";
+}
+
+.grid-container-two{display: grid;
+    padding : 10px;
+    grid-template-columns : 1fr 1fr;
+    column-gap: 10px;
+    background-color:var(--gridBackground) ;
+    height : 6.25rem;
+    border: 2px solid black;
```
Comment (nishasen) [inline components/grids.css]: use variables instead of solid color names

File: components/grids.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.grid-container-three{display: grid;
+    padding : 10px;
+    grid-template-columns : 1fr 1fr 1fr ;
+    column-gap: 10px;
+    background-color:var(--gridBackground) ;
+    height : 6.25rem;
+    border: 2px solid black;
+    border-radius: 10px;
```
Comment (nishasen) [inline components/grids.css:39]: many styles are repeated, you can create a separate class and define the common properties there.

**Review by nishasen — COMMENTED** [review]
The indentation are perfect and easy to understand, css code is also neat. I have suggested some changes please go through it.

File: components/grids.css (hunk 5 above)
Comment (Sameony) [reply components/grids.css:39]: ` border: 2px solid black;`
  `  padding : 10px;`
   ` border-radius: 10px;`
   yes, for instance, a border class can be created with these properties

File: grids.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/components/grids.css">
+    <title>Document</title>
```
Comment (Sameony) [inline grids.html]: Do give a proper title to the webpage

**Review by Sameony — COMMENTED** [review]
Do follow up on these minor adjustments

**Pull request merged**